src/components/Filters/StatefulFilters.tsx

```diff
@@ -14,7 +14,8 @@ import {
   ToolbarGroup,
   ToolbarItem,
   ToolbarSection,
-  SelectOptionObject
+  SelectOptionObject,
+  Button
```

Comment:
Minor: Breaks alphabetization.

Reply:
I think in the UI alphabetization is broken everywhere so perhaps I'll defer that change.